console-extensions.json

```diff
@@ -345,5 +345,88 @@
         "HIDE_STATIC_PIPELINE_PLUGIN_PIPELINERUN_DETAIL_TASKRUNS_TAB"
       ]
     }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "triggers.tekton.dev",
+        "version": "v1beta1",
+        "kind": "EventListener"
+      },
+      "component": { "$codeRef": "triggersList.EventListenersListPage" }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_EVENTLISTENERS_LIST"]
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "triggers.tekton.dev",
+        "version": "v1beta1",
+        "kind": "TriggerBinding"
+      },
+      "component": { "$codeRef": "triggersList.TriggerBindingsListPage" }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_TRIGGERBINDINGS_LIST"]
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "triggers.tekton.dev",
+        "version": "v1alpha1",
+        "kind": "TriggerTemplate"
```

Comment:
<img width="1728" alt="Screenshot 2024-03-25 at 12 52 15 PM" src="https://github.com/openshift-pipelines/console-plugin/assets/102503482/75f3b29b-e37d-4d1c-8842-c84bb2bd81e4">

"version": "v1beta1" version is not added or need to replace v1alpha1 to v1beta1 ?